default/stringtables/en.txt

```diff
@@ -10448,91 +10448,92 @@ SHP_WEAPON_1_2
 Mass Driver 2
 
 SHP_WEAPON_1_2_DESC
-Improves Mass Driver by one level, increasing strength by 1.
+Upgrades [[shippart SR_WEAPON_1_1]] parts on all ships within [[metertype METER_SUPPLY]], increasing shot [[encyclopedia DAMAGE_TITLE]] to 4.
```

Comment:
Think `increasing by X` is preferrable to `increasing to X`, the later is not always correct.